src/library/components/common/external-report-button.js

```diff
@@ -28,13 +28,13 @@ export default class ExternalReportButton extends React.Component {
   }
 
   componentWillReceiveProps (nextProps) {
-    this.setState({disabled: nextProps.isDisabled()})
```

Comment:
This looks wrong. @dougmartin  just changed this to:
`this.setState({disabled: nextProps.isDisabled()})`
I think @pjanik only changed some whitespace so I think the correct merge is:
`this.setState({ disabled: nextProps.isDisabled() })`

Reply:
Ah, I see. Thanks for catching that. I fixed it in the latest commit.